foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/event/BizkeeperProcessingRequestEvent.java

```diff
@@ -14,23 +14,22 @@
  * limitations under the License.
  */
 
-package io.servicecomb.foundation.metrics.performance;
+package io.servicecomb.foundation.metrics.event;
 
-import java.util.List;
+public class BizkeeperProcessingRequestEvent implements MetricsEvent {
```

Comment:
invocation know response is success or fail
so we no need to depend on bizkeeper?

Reply:
Yes,I agree，will calculate latency & tps  by myself